chapters/classes.tex

```diff
@@ -341,8 +341,10 @@ \subsection{Component Variability Prefixes discrete, parameter, constant}\label{
 \begin{itemize}
 \item
   A variable \lstinline!vc! declared with \lstinline!constant!\indexinline{constant} prefix remains constant during transient analysis, with a value that is unaffected by the initialization problem.
+  This is called a constant, or constant variable.
```

Comment:
I suppose this should be treated similar to introduced terminology for expression variability:
```suggestion
  This is called a \firstuse{constant}, or \firstuse{constant variable}\index{constant variable}\index{component variability!constant}.
```